libafl_qemu/librasan/asan/src/tracking/guest_fast.rs

```diff
@@ -0,0 +1,78 @@
+//! # guest
+//! This implementation performs guest memory tracking by use of a hash table
+//! and hence requires no interaction with the guest. Unlike GuestTracking, this
+//! faster variant is unable to detect whether a new allocation overlaps an
+//! existing one (though this should be taken care of by the allocator).
+use core::hash::BuildHasherDefault;
+
+use ahash::AHasher;
```

Comment:
In libafl_bolts we got this guy  that will take one of two hashers, not sure if that's what you would want to use here https://github.com/AFLplusplus/LibAFL/blob/23185b642b43e47156081beb50ae184e83d7a694/libafl_bolts/src/lib.rs#L254

Reply:
Does that support nostd? Is it only for 64 bit architectures? Why would you not want xxh3? Why the 2 configs?

Reply:
no_std + alloc yes.
xxh3 no clue :D Maybe we should drop support. But in any case I'm saying we have a stub function to give you a hasher